README.md

```diff
@@ -174,6 +175,22 @@ for await (const file of unpack(carReader)) {
 }
 ```
 
+### `ipfs-car/unpackStream`
```

Comment:
This needs to be added on line ~91 of the Readme

Reply:
ah, almost, i think that header should be removed, there is no such file!